ansible_base/resource_registry/views.py

```
@@ -122,6 +123,80 @@ def list(self, request, *args, **kwargs):
+                    resource.is_partially_migrated = item["is_partially_migrated"]
+                    update_fields.append("is_partially_migrated")
+
+                if update_fields:
+                    resource.save(update_fields=update_fields)
+
+                if "resource_data" in item:
+                    resource_type = resource.content_type.resource_type
+                    if resource_type.can_be_managed:
+                        try:
+                            resource.update_resource(item["resource_data"], partial=True)
+                        except Exception as e:
```

Ruff (0.15.21)

[warning] Do not catch blind exception: `Exception`

(BLE001)